sdk/communication/azure-communication-common/src/test/java/com/azure/communication/common/implementation/HmacAuthenticationPolicyTests.java

```diff
@@ -245,7 +245,6 @@ public void httpRequestTest() throws MalformedURLException {
 
         HttpRequest request = new HttpRequest(HttpMethod.GET, new URL("http://localhost?id=b93a5ef4-f622-44d8-a80b-ff983122554e"));
         StepVerifier.create(pipeline.send(request))
-            .expectError(RuntimeException.class);
+            .verifyError(RuntimeException.class);
```

Comment:
Why this has been changed?

Reply:
I'm not 100% sure on the difference between `expectError` vs `verifyError`, but this test is supposed to fail at the `http vs https` check. If you debug the code, it was failing somewhere else before.